blogposts/announcing-sourcegraph-3.11.md

```diff
@@ -72,21 +46,29 @@ Language analysis is computationally expensive, so this feature is currently beh
   <p style="text-align: center"><a href="https://vimeo.com/380662673" target="_blank">View on Vimeo</a></p>
 </p>
 
-Sourcegraph 3.11 introduces structural search, a code-aware search syntax that can identify structural patterns in code. This enables advanced code search to specifically match patterns inside code structures such as function parameters and loop bodies.
+Sourcegraph 3.11 introduces [structural search](https://docs.sourcegraph.com/user/search/structural), a code-aware search syntax that can identify structural patterns in code. This enables advanced code search to specifically match patterns inside code structures such as function parameters and loop bodies.
 
 It can be awkward or difficult to match code blocks or nested expressions with regexp. To meet this challenge we’ve introduced a new and easier way to search code that operates more closely on the parse tree of the input using [Comby syntax](https://comby.dev/) for structural matching.
 
-Here is a search query that [matches all the arguments in `fmt.Sprintf` calls](https://sourcegraph.com/search?q=repo%3A%5Egithub%5C.com%2Fsourcegraph%2Fsourcegraph%24++%27fmt.Sprintf%28%3A%5Bargs%5D%29%27+count%3A100&patternType=structural) in the Sourcegraph code base:
+For example, this can be used to [find all the places that `fmt.Sprintf` are unnecessary](https://sourcegraph.com/search?q=repo:sourcegraph/sourcegraph%24+%27fmt.Sprintf%28%22:%5Bstr%5D%22%29%27&patternType=structural):
+
+```text
+patterntype:structural 'fmt.Sprintf(":[str]")'
+```
+
+Or to [identify all the places an `http.Client` is created with a `Transport`](https://sourcegraph.com/search?q=repo:sourcegraph/sourcegraph%24+%22http.Client%7B:%5Ba%5D+Transport:+:%5Bb%5D+:%5Bc%5D%7D%22+count:1000&patternType=structural):
 
 ```text
-patterntype:structural “fmt.Sprintf(:[args])”
+patterntype:structural "http.Client{:[a] Transport: :[b] :[c]}"
 ```
 
-The `:[args]` part is a hole with a descriptive name `args` that matches code. The important part is that this pattern understands that the parentheses are balanced.
+In these examples, the `:[str]` part is a hole with a descriptive name `str` that matches code. The important part is that this pattern understands that the structual pieces (e.g. parentheses, braces) are balanced.
```

Comment:
The above example, and then talking about `str` and explaining things seems a bit unordered. 

> this pattern

Which pattern does this refer to? In the docs this is a specific pattern. In these examples there are two. Maybe `these` patterns.